### PR TITLE
refactor: Use app default logger everywhere 

### PR DESCRIPTION
**Pull request by atomrc**

### CODE REVIEW

File: src/script/calling/entities/FlowEntity.js [hunk 1]
```diff
@@ -60,8 +61,8 @@ z.calling.entities.FlowEntity = class FlowEntity {
     this.conversationId = this.callEntity.id;
     this.messageLog = this.participantEntity.messageLog;
 
-    const loggerName = 'z.calling.entities.FlowEntity';
-    this.callLogger = new z.telemetry.calling.CallLogger(loggerName, this.id, z.config.LOGGER.OPTIONS, this.messageLog);
+    const loggerName = 'FlowEntity';
```
Comment (Yserz) [inline src/script/calling/entities/FlowEntity.js]: If it's not used anywhere else let's make this one inline

File: src/script/calling/entities/ParticipantEntity.js [hunk 2]
```diff
@@ -51,7 +52,7 @@ z.calling.entities.ParticipantEntity = class ParticipantEntity {
     this.sessionId = undefined;
 
     const loggerName = 'z.calling.entities.ParticipantEntity';
-    this.callLogger = new z.telemetry.calling.CallLogger(loggerName, this.id, z.config.LOGGER.OPTIONS, this.messageLog);
+    this.callLogger = new CallLogger(loggerName, this.id, this.messageLog);
```
Comment (Yserz) [inline src/script/calling/entities/ParticipantEntity.js]: If it's not used anywhere else let's make this one inline

File: src/script/entity/message/Message.js [hunk 3]
```diff
@@ -18,6 +18,7 @@
  */
 
 import ko from 'knockout';
+import moment from 'moment';
```
Comment (Yserz) [inline src/script/entity/message/Message.js:21]: why?

Comment (atomrc) [reply src/script/entity/message/Message.js:21]: It was published on the global scope before (the old logger needed it). 
Now every class needs to import it manually

File: test/api/TestFactory.js [hunk 4]
```diff
@@ -30,23 +32,10 @@ window.testConfig = {
...
-
-  const initialLoggerOptions = z.config.LOGGER.OPTIONS;
-  Object.keys(initialLoggerOptions.domains).forEach(domain => {
-    initialLoggerOptions.domains[domain] = logger_level;
-  });
-  initialLoggerOptions.level = logger_level;
-
-  return this;
-};
+window.TestFactory = function() {};
```
Comment (Yserz) [inline test/api/TestFactory.js:38]: Can we remove that?

Comment (atomrc) [reply test/api/TestFactory.js:38]: unfortunalty no, we still do `new TestFactory` everywhere. 
But in the long term, yes we will remove that class entirely :)

**Pull request merged**